component/List.java

```diff
@@ -0,0 +1,110 @@
+///*
+// * Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// *
+// * WSO2 Inc. licenses this file to you under the Apache License,
+// * Version 2.0 (the "License"); you may not use this file except
+// * in compliance with the License.
+// * You may obtain a copy of the License at
+// *
+// * http://www.apache.org/licenses/LICENSE-2.0
+// *
+// * Unless required by applicable law or agreed to in writing,
+// * software distributed under the License is distributed on an
+// * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// * KIND, either express or implied.  See the License for the
+// * specific language governing permissions and limitations
+// * under the License.
+// */
+//package org.ballerinalang.ftp.client.actions;
+//
+//import org.ballerinalang.bre.Context;
+//import org.ballerinalang.ftp.util.FTPUtil;
+//import org.ballerinalang.ftp.util.FtpConstants;
+//import org.ballerinalang.model.types.TypeKind;
+//import org.ballerinalang.model.values.BMap;
+//import org.ballerinalang.model.values.BValue;
+//import org.ballerinalang.model.values.BValueArray;
+//import org.ballerinalang.natives.annotations.BallerinaFunction;
+//import org.ballerinalang.natives.annotations.Receiver;
+//import org.slf4j.Logger;
+//import org.slf4j.LoggerFactory;
+//import org.wso2.transport.remotefilesystem.RemoteFileSystemConnectorFactory;
+//import org.wso2.transport.remotefilesystem.client.connector.contract.FtpAction;
+//import org.wso2.transport.remotefilesystem.client.connector.contract.VFSClientConnector;
+//import org.wso2.transport.remotefilesystem.exception.RemoteFileSystemConnectorException;
+//import org.wso2.transport.remotefilesystem.impl.RemoteFileSystemConnectorFactoryImpl;
+//import org.wso2.transport.remotefilesystem.message.RemoteFileSystemBaseMessage;
+//import org.wso2.transport.remotefilesystem.message.RemoteFileSystemMessage;
+//
+//import java.util.HashMap;
```

Comment:
Why did we comment the entire code instead of removing the file?

Reply:
File was not removed as it was in the process of being migrated. Removed in #54.